pkg/controllers/interruption/metrics.go

```diff
@@ -25,6 +25,7 @@ const (
 	interruptionSubsystem  = "interruption"
 	messageTypeLabel       = "message_type"
 	actionTypeLabel        = "action_type"
+	nodePoolLabel          = "nodepool"
```

Comment:
I think you figured this out, but you can drop this line since you have an upstream definition for this e.g. `metrics.NodePoolLabel`

Reply:
Agreed, updated the PR removing the label in favor of the upstream.